src/components/settings/SettingsSSOTab/SSOFormWorkflowConfig.tsx

```diff
@@ -105,11 +105,24 @@ export const SSOFormWorkflowConfig = ({ enterpriseId, setConfigureError }) => {
   }: FormWorkflowHandlerArgs<SSOConfigFormContextData>) => {
     let err = null;
 
-    // Accurately detect if form fields have changed
-    if (!formFieldsChanged) {
-      // Don't submit if nothing has changed
+    // Accurately detect if form fields have changed or there's and error in existing record
+    let isErrored;
+    if (formFields?.uuid) {
+      const ssoRecord =
+        await LmsApiService.fetchEnterpriseSsoOrchestrationRecord(
+          formFields?.uuid
+        ).catch(() => {
+          return { data: {} };
+        });
```

Comment:
@alex-sheehan-edx I just gone through code again, `formFields` is supposed to have many other attributes given [here](https://github.com/edx/frontend-app-admin-portal/blob/5de18f9dc22a67cdc4a745c56430014b09de2c74/src/components/settings/SettingsSSOTab/SSOFormWorkflowConfig.tsx#L47).

So, can we use any keys from `formFields` instead of making this separate API call?